lang/cs/Org.Apache.REEF.Common/Runtime/Evaluator/EvaluatorRuntime.cs

```diff
@@ -241,6 +250,11 @@ private void OnException(Exception e)
             }
         }
 
+        private void UnhandledExceptionHandler(object sender, UnhandledExceptionEventArgs e)
```

Comment:
This catches Exceptions for all children threads, so Tasks should not have the need to bind a separate instance for this.